extras/bindings/c/include/precice/SolverInterfaceC.h

```diff
@@ -33,6 +33,23 @@ void precicec_createSolverInterface(
     int         solverProcessIndex,
     int         solverProcessSize);
 
+/**
+ * @param[in] participantName Name of the participant using the interface. Has to
+ *        match the name given for a participant in the xml configuration file.
+ * @param[in] configurationFileName Name (with path) of the xml configuration file.
+ * @param[in] solverProcessIndex If the solver code runs with several processes,
+ *        each process using preCICE has to specify its index, which has to start
+ *        from 0 and end with solverProcessSize - 1.
+ * @param[in] solverProcessSize The number of solver processes using preCICE.
+ * @param[in] communicator A pointer to an MPI_Comm to use as MPI_COMM_WORLD.
```

Comment:
We are using the given communicator as a base communicator.

```suggestion
 * @param[in] communicator A pointer to an MPI_Comm to use as communicator.
```

Reply:
This was a copy of the equivalent C++ method documentation. Fixed in both.